src/sass/main.scss

```diff
@@ -1,15 +1,52 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
-
-#color-form {
+body {
+  height: 100vh;
+  margin: 0;
   font-family: 'Roboto', sans-serif;
 }
 
+main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.color-form {
+  margin-bottom: 2rem;
+  min-width: 95vw;
+  margin: 2rem auto;
+
+  // if we do want to "center" the form, can add the media queries from below here
+}
+
+.grid-container {
+  --width: 95vw;
+  --grid: calc(var(--width) / var(--size));
+  text-align: center;
+  padding-bottom: 2rem;
+
+  @media (min-width: 50rem) {
+    --width: 80vw;
+  }
+  
+  @media (min-width: 100rem) {
+    --width: 40vw;
+  }
+
+  @media (orientation: landscape) {
```

Comment:
Also fancy! Is there a reason you went with this media query versus a `min-height`? Just thinking `min-height` would be more broad to cover use cases like a desktop window that is wide and short or could allow you to account fora landscape phone vs tablet.